packages/components/src/styles/components/side-nav/main.scss

```diff
@@ -7,7 +7,26 @@
 // SIDE-NAV
 //
 
+// TODO: Create app-header tokens:
+:root {
+  --token-hds-app-header-height: 60px;
+  --token-hds-app-header-home-link-padding: 4px;
+}
+
+:root {
+  // TODO: Create sidenav token
+  --hds-app-sidenav-headerless-width-minimized: 16px;
+
+  // TODO: Create app-header token:
+  --hds-app-header-height: 59px;
```

Comment:
I will add these tokens to JSON but wanted to confirm if naming pattern I used is correct?

Reply:
While here is `59px` while above is `60px`? 🤔

Reply:
Oops, I will fix, thanks